plugins/com.google.cloud.tools.eclipse.sdk.test/src/com/google/cloud/tools/eclipse/sdk/internal/CloudSdkProviderTest.java

```diff
@@ -48,4 +58,153 @@ public void testSetPreferenceInvalid() throws Exception {
       // ignore
     }
   }
+  
+  private static class MockPreferences implements IPreferenceStore{
```

Comment:
Mockito didn't work. We cab probably make it work, but for now this gets us moving again.